OrbitGl/App.cpp

```diff
@@ -1009,7 +1009,7 @@ void OrbitApp::OnRemoteProcess(const Message& a_Message) {
     m_ModulesDataView->SetProcess(remote_process);
     // Is this needed?
     Capture::SetTargetProcess(remote_process);
-    FireRefreshCallbacks();
+    FireRefreshCallbacks(DataViewType::MODULES);
```

Comment:
Don't other views need to be updated, albeit not all of them? For example, when selecting a new process, the loaded functions are cleared and hence the "functions" tab's view is cleared. Does that still happen?

Reply:
Right.. I will let it update everything here then.

Reply:
done.. functions do not work either way but there is a separate bug for it.